deploy/crds/metal3.io_baremetalhosts_crd.yaml

```diff
@@ -436,6 +436,20 @@ spec:
               - ID
               - state
               type: object
+            softPowerOff:
```

Comment:
Instead of adding these new fields, could we use the existing power status field? It would only allow us to try using soft power off one time, but I think that's OK.